### ISSUE TITLE
Image Render

### ISSUE THREAD

**Issue by azacchino** [oldrev/maltreport#18]
Hello, I tried to create a document (odt) with an embedded image. In the image, just replace the name with "$var". When you try to replace it with the context, the file is broken. How do I associate the image I want to replace with the context?

Using NuGet Version ( 2.1.1 ) 

```  C#

System.IO.MemoryStream ms = new MemoryStream();
bmp.Save(ms, System.Drawing.Imaging.ImageFormat.Png);
ms.GetBuffer();
Image im = new Image("png", ms.GetBuffer());

var renderContext = new Dictionary<string, object>()
{
       { "var", im },
}

OdfTemplate a = new OdfTemplate();
a.Load(path);
a.Compile();
IDocument b = a.Render(renderContext);
b.Save(@"C:\temp\simpleImageMap.ods");
``` 
Regargs

**Comment by oldrev** [comment]
Hello:

Sorry for the late response. Image rendering with MaltReport 2.1.1 has some serious issues, please try 3.x preview, it's stable enough to works with ODT/ODS. 

MaltReport 3.x supports .NET FX 4.6 and .NET Core both.

Please see the [HelloWorld project](https://github.com/oldrev/maltreport/tree/master/samples/MaltReport.HelloWorld) as an image-rendering sample in in the source repository.

**Comment by azacchino** [comment]
Hi! Thanks for the response. The 3.x branch hasn't tag. ¿the master branch is the 3.x?

**Comment by oldrev** [comment]
Yes, the master branch is 3.x, or you can install lib from nuget:
https://www.nuget.org/packages/MaltReport2/3.0.0-beta-7

**Comment by Franklin89** [comment]
How does MaltReport know which image it has to replace?
I want two images in my report. How do I have to add the image to the template?
